README.md

```diff
@@ -28,7 +28,7 @@ To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
 
-- Knowledge on basic html and css
+- Knowledge on basic html5 and css3
 - Text Editor (VScode, Atom, Sublime text, etc)
 - Basic knowledge of node.js
```

Comment:
- [ ] Please the following weird behavior I record in this video: -

https://user-images.githubusercontent.com/74744787/165965790-870f94e1-e57e-46fa-b75f-6f2ae1d86cda.mp4

- [ ] When the user completes an action then clears all the completed actions all the tasks on the to-do list are dissapered 
- [ ] When the user completes an action then reloads the page the check and the completion styles are removed please make sure to fix this

Reply:
Hi, @MeqdamAlqudah thanks for the review. I have fixed these issues and have already gotten approval. According to student success I am supposed to resubmit this activity for approval since it has already been approved. So am confused as to why you are asking me to do something I have already done and got approval for i.

NB: am repeating this module. All projects were approved

Reply:
Hi, @MeqdamAlqudah thanks for the review. I have fixed these issues and have already gotten approval. According to student success I am supposed to resubmit this activity for approval since it has already been approved. So am confused as to why you are asking me to do something I have already done and got approval for i.

NB: am repeating this module. All projects were approved

Reply:
Hi, @MeqdamAlqudah thanks for the review. I have fixed these issues and have already gotten approval. According to student success, I am supposed to resubmit this activity for approval since it has already been approved. So am confused as to why you are asking me to do something I have already done and got approval for i.

NB: am repeating this module. All projects were approved